doc/install/install-vm-cloud.rst

```diff
@@ -57,9 +57,13 @@ To install QEMU, execute the following:
 	[main]
 	enabled = 1
 
+.. note:: ceph-extras is not needed for EL-7 variants and Fedora.  It is necessary
+   for EL-6.
+
 #. Create a ``/etc/yum.repos.d/ceph-extras.repo`` file with the following 
-   contents, and replace ``{distro}`` with your Linux distribution. Follow
-   the ``baseurl`` path below to see which distributions Ceph supports:: 
+   contents, and replace ``{distro}`` with your Linux distribution. Only
+   older distributions require ceph-extras. Follow the ``baseurl`` path
```

Comment:
Good point.  Would you be happier if I just dropped that line entirely?  I already define what versions require/don't require ceph-extras in the ..note immediately before this.

Or I can just say something like "Only -EL6 based distributions and older...."

If I do the latter, is the "and older" bit necessary?  I don't actually know if we can install on EL5.  I don't think we can.  So maybe it is better to just say "Only -EL6 based distributions require ceph-extras"